packages/server/src/workers/utils.ts

```diff
@@ -108,6 +108,17 @@ export async function createAuditEvent(
   });
 }
 
+export function validDeleteInteraction(subscription: Subscription): boolean {
```

Comment:
For functions that return booleans, we try to use `is` prefixes.  So either `isValidDeleteInteraction` or just `isDeleteInteraction`